guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md

```diff
@@ -96,6 +106,44 @@ If the operation is successful, the response contains the code of the selected p
 }
 ```
 
+### Set payment method and place order {#setPaymentMethodAndPlaceOrder}
```

Comment:
I've just realized that this mutation is not a part of 2.3.2 version. I would suggest splitting this PR into two separate PRs and making the PR with information about `setPaymentMethodAndPlaceOrder` mutation to the develop branch.

Reply:
I've created a separate PR https://github.com/magento/devdocs/pull/5000 to fix 
1. Remove `address_type` from `shipping_addresses` and `billing_addresses`.
2. Remove `label` from `selected_shipping_method`

Reply:
Thank you. The only change left is to create a PR to the `develop` branch.

Reply:
@rogyar I will reset the target branch for this PR into the `develop` branch immediately after the next time we sync `master` and `develop`.